specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimloggers.json

```diff
@@ -352,7 +355,8 @@
           "type": "string",
           "description": "Logger type.",
           "enum": [
-            "azureEventHub"
+            "azureEventHub",
```

Comment:
new value added to the enum? This should require new api-version.

Reply:
@solankisamir thoughts?

Reply:
We have modeled our enum as string to avoid this breaking change (based on guidance from SDK Team). Are we still required to create a new version?

---
In reply to: [161912170](https://github.com/Azure/azure-rest-api-specs/pull/2250#discussion_r161912170) [](ancestors = 161912170)

Reply:
cool, if SDK has agreed.

Reply:
I don't see the presence of

Reply:
Yes, this is fine `"modelAsString": true,` is present, so that means existing clients can take the value, and there is no `Enum` created, so we're all good.